src/EnergyPlus/HVACManager.cc

```diff
@@ -1845,7 +1783,7 @@ void SimHVAC(EnergyPlusData &state)
         }
 
         if (state.dataCoilCooingDX->stillNeedToReportStandardRatings) {
-            if (!state.dataGlobal->ZoneSizingCalc && !state.dataGlobal->SysSizingCalc && !state.dataGlobal->WarmupFlag) {
+            if (!state.dataGlobal->WarmupFlag) {
```

Comment:
[src/EnergyPlus/HVACManager.cc:1848]:(style),[knownConditionTrueFalse],Condition '!state.dataGlobal->ZoneSizingCalc' is always true

Reply:
:+1: love to see it cleaned out.